data/drules_proto.txt

```diff
@@ -67946,10 +67963,26 @@ cont {
   element {
     scale: 16
     lines {
-      width: 1.0
+      width: 5.0
+      color: 10592406
+      dashdot {
+      }
```

Comment:
Hm, we should check drules generator, Why empty dashdot ..

Reply:
yeah there are empty dashdots at some existing elements like `highway-motorway-bridge` z13; and even seemingly conflicting `lines`
```
  15291   element {
  15292     scale: 13
  15293     lines {
  15294       width: 2.8
  15295       color: 16742950
  15296       priority: 1446
  15297       cap: BUTTCAP
  15298     }
  15299     lines {
  15300       width: 6.0
  15301       color: 15592411
  15302       dashdot {
  15303       }
  15304       priority: 1280
  15305       cap: BUTTCAP
  15306     }
  15307     lines {
  15308       width: 6.4
  15309       color: 8421504
  15310       dashdot {
  15311       }
  15312       priority: 1270
  15313       cap: BUTTCAP
  15314     }
```